lua/lib/reflection.lua

```diff
@@ -128,7 +129,7 @@ end
 
 --- quantize playback
 -- @tparam float q defaults to 1/48
--- (should be at least 1/96)
+-- (should be at least 1/48)
```

Comment:
why this change? it seems like 1/96 should behave as expected?

Reply:
oh! the steps don't seem to execute with 1/96, couldn't tell if it was intentional. playback starts and eventually ends with this test script, but no step actions are executed:

```lua
_r = require 'reflection'
my_pattern = _r.new()

function init()
  my_pattern.process = process_pat
  my_pattern.event = {
    [1] = {[1] = {st = 'hello beat 1'}},
    [96] = {[1] = {st = 'hello beat 2'}},
    [192] = {[1] = {st = 'hello beat 3'}},
    [288] = {[1] = {st = 'hello beat 4'}},
  }
  my_pattern.count = 3
  my_pattern:set_length(4)
  my_pattern:set_quantization(1/96)
  clock.run(
    function()
      clock.sync(4)
      my_pattern:begin_playback()
    end
  )
end

function my_pattern.start_callback()
  print('playback started', clock.get_beats())
end

function my_pattern.end_of_rec_callback()
  print('recording finished', clock.get_beats())
end

function my_pattern.end_of_loop_callback()
  print('loop ended', clock.get_beats())
end

function my_pattern.end_callback()
  print('playback ended', clock.get_beats())
end

function process_pat(e)
  print(e.st, clock.get_beats())
end
```

Reply:
mmm:
https://github.com/monome/norns/blob/03f158816346405c4b595f7602ebd66216be2c87/lua/lib/reflection.lua#L209

sets `q` to `1` when `self.quantize` is `1/96`, which fails this check later:

https://github.com/monome/norns/blob/03f158816346405c4b595f7602ebd66216be2c87/lua/lib/reflection.lua#L235

Reply:
oh that's bad math on my part ... still not thinking about it!! but excited to dig in later

Reply:
hoho! yeah, `if q ~= 1 and self.step % q ~= 1 then goto continue end` fixes it up!